src/main/resources/interface-specification.yml

```diff
@@ -4011,6 +4011,105 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
+  /purposes/{purposeId}/versions/{versionId}/archive:
+    parameters:
+      - $ref: '#/components/parameters/CorrelationIdHeader'
+      - $ref: '#/components/parameters/IpAddress'
+      - name: purposeId
+        in: path
+        required: true
+        schema:
+          type: string
+          format: uuid
+      - name: versionId
+        in: path
+        required: true
+        schema:
+          type: string
+          format: uuid
+    post:
+      tags:
+        - purposes
+      description: archives the purpose version by id
+      summary: Archive Purpose Version
+      operationId: archivePurposeVersion
+      responses:
+        '200':
+          description: Purpose Version Archived
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CompactPurposeVersion'
```

Comment:
let's pause this PR until [this one](https://github.com/pagopa/interop-be-backend-for-frontend/pull/116/files#diff-17a3d713b849787a0ed57552f2b367a47eedf4cef81a6ba025d0060ac107d213R4041) has been marged, so you can return the same model